e2e/docker/docker.go

```diff
@@ -1157,6 +1193,60 @@ func (c ctx) testDockerCMDQuotes(t *testing.T) {
 // Check that the USER in a docker container is honored under --oci mode
 func (c ctx) testDockerUSER(t *testing.T) {
 	dockerURI := "docker://sylabsio/docker-user"
+	dockerfile := filepath.Join("..", "test", "defs", "Dockerfile.customuser")
+	tmpdir, tmpdirCleanup := e2e.MakeTempDir(t, "", "dockerfile-build-USER-", "temp dir for OCI-SIF images")
+	t.Cleanup(func() {
+		if !t.Failed() {
+			tmpdirCleanup(t)
+		}
+	})
+	userBuiltOCISIF := filepath.Join(tmpdir, "docker-user.oci.sif")
+	c.env.RunSingularity(
+		t,
+		e2e.AsSubtest("df build"),
+		e2e.WithProfile(e2e.UserProfile),
+		e2e.WithCommand("build"),
+		e2e.WithArgs("--oci", userBuiltOCISIF, dockerfile),
+		e2e.ExpectExit(0),
+	)
+	rootBuiltOCISIF := filepath.Join(tmpdir, "rootbuilt-docker-user.oci.sif")
+	c.env.RunSingularity(
+		t,
+		e2e.AsSubtest("df build"),
```

Comment:
Don't use the same subtest name twice. Go suffixes them with numbers then, and it's a bit confusing.

`user df build` and `root df build` should be fine.